packages/forma-36-react-components/src/components/EditorToolbar/EditorToolbarDivider/EditorToolbarDivider.stories.tsx

```diff
@@ -18,6 +18,6 @@ export default {
   title: 'Components/EditorToolbar/EditorToolbarDivider',
 } as Meta;
 
-export const Default: Story<EditorToolbarDividerProps> = ({ ...args }) => {
+export const Default = ({ ...args }: EditorToolbarDividerProps) => {
```

Comment:
We're losing the `Story` type by changing this, let me see if I can dig up the GitHub comment I got it from.

Reply:
Do we actually need this type here?

Reply:
I'm not sure to be honest, I think it could potentially add and enhance other types for us.

Reply:
https://github.com/storybookjs/storybook/blob/next/docs/snippets/react/page-story.ts.mdx